streams/readable-byte-streams/enqueue-with-detached-buffer.window.js

```diff
@@ -5,7 +5,11 @@ promise_test(async t => {
     pull: t.step_func((controller) => {
       const buffer = controller.byobRequest.view.buffer;
       // Detach the buffer.
-      postMessage(buffer, '*', [buffer]);
+      if(typeof postMessage === 'function') {
+        postMessage(buffer, '*', [buffer]);
+      } else {
+        structuredClone(buffer, { transfer: [buffer] });
```

Comment:
Left `postMessage` in case there's a platform running this test that doesn't yet support `structuredClone`

Reply:
Can we have a separate test with structuredClone below, and add an `assert_implements_optional` here so that it can pass in Deno? With the current PR Deno and browsers will have different code paths which wouldn't be ideal.